tests/user_type/byte_address_buffer_0.spv.yaml

```diff
@@ -101,7 +101,7 @@ all_descriptor_bindings:
     accessed: 1
     uav_counter_id: 4294967295
     uav_counter_binding:
-    ByteAddressBuffer offsets: [4, 5, 11, 13]
+    ByteAddressBuffer offsets: [13, 5, 11, 4]
```

Comment:
little concerned why this was flipped here, but not in `user_type/byte_address_buffer_3.spv.yaml`

Reply:
less concern because the order wasn't important/required